.github/workflows/protobuf.yml

```diff
@@ -32,10 +32,9 @@ jobs:
           fi
         # yamllint enable
 
-  # add this back when we start using versioning
-  # breakage:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - name: check-breakage
-  #       run: make proto-check-breaking
+  breakage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: check-breakage
+        run: make proto-check-breaking
```

Comment:
[blocking] I can't comment on this line directly but can we either update or delete the top line of this file. IMO seems safe to delete.

Ref: https://github.com/celestiaorg/celestia-app/pull/2671/files#diff-a44d106fd1ff0127f62ef1b829e39ff1e4697db877213026cd87e57e7728161fR1